policyengine_us/parameters/gov/states/az/tax/income/property_tax_credits/income_sources.yaml

```diff
@@ -0,0 +1,12 @@
+description: Arizona accounts for the following income sources for the property tax credit.
+values:
+  2021-01-01:
+    - capital_gains_excluded_from_taxable_income
+    - tax_exempt_interest_income
+    - public_pension_income
+    - adjusted_gross_income
+metadata:
+  unit: list
+  reference:
+    - title: 43-1072. Earned credit for property taxes; residents sixty-five years of age or older(H-6)
```

Comment:
add tax form reference if there is one, if there is not add comment: # Income not mentioned in the tax form